### ISSUE TITLE
TypeError in make_transfers.py

### ISSUE THREAD

**Issue by Tdarnell** [alan-turing-institute/AIrsenal#478]
I get the following error whilst running pipeline and making transfers (following a free hit last week).

I am running WSL Ubuntu 20.04 with a miniconda environment. 

```
Optimization complete..
Applying suggested transfers...
Getting starting squad for 1760571
Getting starting squad for 1760571
Traceback (most recent call last):
  File "/home/user/miniconda3/envs/airsenalenv/bin/airsenal_run_pipeline", line 8, in <module>
    sys.exit(run_pipeline())
  File "/home/user/miniconda3/envs/airsenalenv/lib/python3.8/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/user/miniconda3/envs/airsenalenv/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/user/miniconda3/envs/airsenalenv/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/user/miniconda3/envs/airsenalenv/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/user/miniconda3/envs/airsenalenv/lib/python3.8/site-packages/airsenal/scripts/airsenal_run_pipeline.py", line 143, in run_pipeline
    transfers_ok = make_transfers(fpl_team_id)
  File "/home/user/miniconda3/envs/airsenalenv/lib/python3.8/site-packages/airsenal/scripts/make_transfers.py", line 264, in make_transfers
    post_transfer_bank = deduct_transfer_price(pre_transfer_bank, priced_transfers)
  File "/home/user/miniconda3/envs/airsenalenv/lib/python3.8/site-packages/airsenal/scripts/make_transfers.py", line 42, in deduct_transfer_price
    gain = [
  File "/home/user/miniconda3/envs/airsenalenv/lib/python3.8/site-packages/airsenal/scripts/make_transfers.py", line 43, in <listcomp>
    transfer["selling_price"] - transfer["purchase_price"]
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'
```

I will not be able to take a look at this bug until next week, so reporting in case others get the chance.

**Comment by jack89roberts** [comment]
Thanks for flagging the issue @Tdarnell . Just to double-check - this only comes up when applying the transfers? The optimisation itself runs normally (and prints a proposed transfer strategy correctly etc.)?

**Comment by Tdarnell** [comment]
That's correct, the optimisation results print correctly first

**Comment by Tdarnell** [comment]
Interestingly I get the same error for this week too @jack89roberts 

`[{'element_out': 578, 'selling_price': 59, 'element_in': 701, 'purchase_price': 61}, {'element_out': 437, 'selling_price': None, 'element_in': 368, 'purchase_price': 43}] `

Team ID is 1760571

Could this be an issue in the fetcher? 

The data from the API is as follows: 
```{"element_in":6,"element_in_cost":83,"element_out":579,"element_out_cost":124,"entry":1760571,"event":28,"time":"2022-03-04T12:22:33.178835Z"},{"element_in":233,"element_in_cost":132,"element_out":240,"element_out_cost":83,"entry":1760571,"event":28,"time":"2022-03-04T12:22:33.174250Z"}```
https://fantasy.premierleague.com/api/entry/1760571/transfers/

**Comment by jack89roberts** [comment]
Ah right yes it could be, could you see if you have the same problem if you install and run AIrsenal on the latest `develop` branch (see [the readme](https://github.com/alan-turing-institute/AIrsenal#issues-and-development) for instructions if you've not done that before)? We have a fix there for #475 but it's not merged yet.

**Comment by Tdarnell** [comment]
> Ah right yes it could be, could you see if you have the same problem if you install and run AIrsenal on the latest `develop` branch (see [the readme](https://github.com/alan-turing-institute/AIrsenal#issues-and-development) for instructions if you've not done that before)? We have a fix there for #475 but it's not merged yet.

That branch works :)
